modules/lvgl/input/lvgl_pointer_input.c

```diff
@@ -129,6 +130,46 @@ static void lvgl_pointer_process_event(struct input_event *evt, void *user_data)
 
 int lvgl_pointer_input_init(const struct device *dev)
 {
+	const struct lvgl_pointer_input_config *lv_pointer_cfg = dev->config;
+	struct lvgl_pointer_input_data *lv_pointer_data = dev->data;
+	const struct device *disp_dev = lv_pointer_cfg->common_config.display_dev;
+	struct lvgl_disp_data *lv_disp_data;
+	lv_display_t *lv_disps[DT_ZEPHYR_DISPLAYS_COUNT];
+	lv_display_t *lv_disp = NULL;
+
```

Comment:
Maybe this should be part of `lvgl_common_input.h` instead, because you also need to link the other pseudo devices to a certain screen. Something like:
```
lv_display_t* lvgl_input_get_display(const struct device* dev)
```